tests/test_self_update.cpp

```
@@ -226,6 +233,102 @@ TEST_F(InstallStdlibTest, OldLayoutArchiveInstallsToLibStd) {
     EXPECT_TRUE(fs::exists(dest_std / "builtins.ry"));
 }
 
+TEST_F(InstallStdlibTest, ReturnsFalseWhenNoStdlibInArchive) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)